README.md

```diff
@@ -77,6 +77,25 @@ Connect to Atlas Stream Processing instances and develop stream processors using
 
 ![Atlas Stream Processing Playground](resources/screenshots/atlas-stream-processing.png)
 
+### MongoDB Copilot Participant
+
+Use a natural language to interact with your clusters and generate MongoDB-related code through GitHub Copilot Chat in VS Code.
+
+_Note: To use the MongoDB Participant, you must have the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension. By using Copilot Chat you agree to [GitHub Copilot chat preview terms](https://docs.github.com/en/copilot/responsible-use-of-github-copilot-features/responsible-use-of-github-copilot-chat-in-your-ide). Find more details about the MongoDB GenAI Features in the [FAQ](https://www.mongodb.com/docs/generative-ai-faq/)._
+
+#### How to use the MongoDB Participant
+
+1. Enter `@MongoDB` in the chat input field to start a conversation with the MongoDB Participant.
+
+![MongoDB Participant](resources/screenshots/mongodb-participant.png)
+
+2. Start typing `/` in the chat window to get the list of available chat participant commands.
+
+![MongoDB Participant Commands](resources/screenshots/mongodb-participant-commands.png)
+
+- `/docs` is a participant command, which finds answers to coding-related questions in the [MongoDB documentation](https://www.mongodb.com/docs/).
+- `/query` is a participant command, which generates queries from a natural language against a connected MongoDB cluster. It generates both queries and aggregation depending on the complexity of the request. It utilizes schema to reduce model hallucinations. It providing code actions to either open the query in a playground or directly run the query from the Copilot chat interface.
```

Comment:
```suggestion
- `/query` is a participant command that generates MongoDB queries from natural language to be used with a connected MongoDB cluster. It generates both queries and aggregations depending on the complexity of the request. It utilizes schema to reduce model hallucinations. It provides a code action to open generated code in a playground and an action to directly run the code from the Copilot chat interface.
```
/query can be used for more than queries and aggregations, so we might want to mention that rather than suggesting its only for queries and aggregations. That being said I think this is fine as is (with some typos fixed).

Reply:
Feel free to suggest additions if you have something in mind!